scripts/override-agent-apm-server.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Takes as arguments the apm-server binary path and the apm-server binary file name; 
+# and uncompresses it in the right folder inside the elastic-agent container (must be running)
+
+# Path and file name are separated arguments because the former comes from user input, and the later does not
+# This is because if the file name comes from the user and is not what elastic agent expects, 
+# it will be very hard to debug why it doesn't work
+
+elastic_agent_id=$(docker ps -qf "name=elastic-agent")
+sha=$(docker inspect "$elastic_agent_id" | grep org.label-schema.vcs-ref | awk -F ": \"" '{print $2}' | head -c 6)
+dst=/usr/share/elastic-agent/data/elastic-agent-"$sha"/install/${2%".tar.gz"}
+
+echo "copying ${1}/${2} to $elastic_agent_id:$dst"
+docker exec $elastic_agent_id mkdir $dst
+docker cp "${1}/${2}" "$elastic_agent_id":"$dst"
+
+echo "uncompressing $dst/${2}"
```

Comment:
```suggestion
echo "uncompressing $dst"
```

Reply:
this is intentional, see https://github.com/elastic/apm-integration-testing/pull/1030/files#diff-dabc3041993c7ed3834455aa519c578a5902bc8196fb79f31bac97cc0b237f56R7